src/styles/main.css

```diff
@@ -49,6 +49,9 @@
 .tile-hover-text-position {
   bottom: 1.5rem;
 }
+span.absolute.left-0.right-0.top-0.text-center.text-xl.text-green-400.custom-radio-buttons {
```

Comment:
Something less brittle going forward may be to upgrade Tailwind to the latest version and take advantage of their JIT feature. Then you could delete this and specify the following in your markup:

```
top-[1.6rem]
```